katalog/tests/istio-operator/requirements.sh

```diff
@@ -60,3 +60,19 @@ load ./../helper
   run test
   [ "$status" -eq 0 ]
 }
+
+@test "wait for cert-manager requirements to be running" {
+  info
+  test(){
+    retry_counter=0
+    max_retry=30
+    while kubectl get pods -n cert-manager | grep -ie "\(Pending\|Error\|CrashLoop\|ContainerCreating\|PodInitializing\|Init:\)" >&2
```

Comment:
I guess this works, but maybe it would be better to use `kubectl wait`? https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait

Reply:
Hi Ramiro, this script is never called, instead I modified as required the katalog/tests/istio-operator/istio-requirements.sh script used in the pipeline.